code/game/objects/items/weapons/switchtool.dm

```diff
@@ -161,24 +161,27 @@
 	var/list/potential_modules = list()
 	for(var/module in stored_modules)
 		if(stored_modules[module])
-			potential_modules += get_module_name(module)
+			potential_modules += addtext(get_module_name(module),"(",(stored_modules[module]).name,")")
```

Comment:
There's an easier to read way of concatenating strings. Square brackets lets you execute some code and return their results as a string while in a string itself.

potential_modules += "[get_module_name(module)] ([stored_modules[module].name])" for instance for this.